packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -111,16 +120,25 @@ export class OpenId4VciHolderService {
 
     this.logger.debug('Full server metadata', metadata)
 
+    const credentialOfferPayload = client.credentialOffer.credential_offer
+
+    const offeredCredentialsData = isCredentialOfferV1Draft13(credentialOfferPayload)
+      ? credentialOfferPayload.credential_configuration_ids
+      : credentialOfferPayload.credentials
+
+    const offeredCredentialConfigurations = getOfferedCredentials(
+      offeredCredentialsData,
+      (credentialIssuerMetadata.credentials_supported as OpenId4VciCredentialSupportedWithId[] | undefined) ??
+        credentialIssuerMetadata.credential_configurations_supported
+    )
+
     return {
       metadata: {
         ...metadata,
-        credentialIssuerMetadata: issuerMetadata,
+        credentialIssuerMetadata: credentialIssuerMetadata,
       },
-      credentialOfferPayload,
```

Comment:
I agree, we need to keep the minimum amount  (no duplication), but if it's breaking the existing API interface we should keep it. Sorry If i suggested otherwise before (it's hard to see in a PR review what is causing breaking changes to the API)